frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.styled.tsx

```diff
@@ -108,9 +108,6 @@ function getParameterPanelBackgroundColor(theme?: DisplayTheme) {
   if (theme === "night") {
     return color("bg-black");
   }
-  if (theme === "transparent") {
```

Comment:
This was a dead code. This function is only called when `isSticky` is true.

https://github.com/metabase/metabase/blob/fc0c4a9a7aef35fc01597b7abea6dd454952eb6e/frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.styled.tsx#L140-L151

And when `theme="transparent"`, `isSticky` will be false (We'll fix this at later task making sticky filter works with transparent background)
https://github.com/metabase/metabase/blob/fc0c4a9a7aef35fc01597b7abea6dd454952eb6e/frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.tsx#L153-L156